apps/condo/domains/user/access/_internalResetSMSDayLimitCountersService.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Generated by `createservice user._internalResetSMSDayLimitCountersService`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function can_internalResetSMSDayLimitCounters ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return true
```

Comment:
There's a lot of people in support right now, maybe we should add directAccessRight here instead?

Reply:
Who will be the manager of the new UserRightSet model (connect it to the User models and disconnect sometimes)?

Reply:
The head of support I guess